qiskit_experiments/library/randomized_benchmarking/clifford_utils.py

```diff
@@ -64,67 +79,32 @@ class CliffordUtils:
         (2, 2, 3, 3, 3, 3, 4, 4),
         (2, 2, 3, 3, 4, 4),
     ]
+    GENERAL_CLIFF_LIST = ["id", "h", "sdg", "s", "x", "sx", "sxdg", "y", "z", "cx"]
+    TRANSPILED_CLIFF_LIST = ["sx", "rz", "cx"]
```

Comment:
This assumes a particular IBM-ish backend. Other providers may use different basis set depending on their hardware architecture, or, even us may want to use different set, e.g. "ecr" rather than "cx" which are locally equivalent.

Reply:
It is difficult to make the code efficient without optimizing it for a specific set of basis gates.
For non-IBM backend, one can change the basis gates, and pre-generate the data files (there is a code for this in `generate_transpiled_circuits.py`).
As for an `ecr` gate, it is currently not one of the gates that the Clifford class can handle: 
https://qiskit.org/documentation/stubs/qiskit.quantum_info.Clifford.html

Reply:
Right, but Qiskit is backend agnostic. This violates very important policy of Qiskit.